pwa/pages/_app.tsx

```diff
@@ -2,223 +2,182 @@ import Layout from '@components/common/Layout';
+          media="(device-width: 375px) and (device-height: 812px) and (-webkit-device-pixel-ratio: 3) and (orientation: portrait)"></link>
+        <link
+          rel="apple-touch-startup-image"
+          href="/pwa_icons/apple-splash-2436-1125.jpg"
+          media="(device-width: 375px) and (device-height: 812px) and (-webkit-device-pixel-ratio: 3) and (orientation: landscape)"></link>
+        <link
+          rel="apple-touch-startup-image"
+          href="/pwa_icons/apple-splash-1242-2688.jpg"
+          media="(device-width: 414px) and (device-height: 896px) and (-webkit-device-pixel-ratio: 3) and (orientation: portrait)"></link>
+        <link
+          rel="apple-touch-startup-image"
+          href="/pwa_icons/apple-splash-2688-1242.jpg"
+          media="(device-width: 414px) and (device-height: 896px) and (-webkit-device-pixel-ratio: 3) and (orientation: landscape)"></link>
+        <link
+          rel="apple-touch-startup-image"
+          href="/pwa_icons/apple-splash-828-1792.jpg"
+          media="(device-width: 414px) and (device-height: 896px) and (-webkit-device-pixel-ratio: 2) and (orientation: portrait)"></link>
+        <link
+          rel="apple-touch-startup-image"
+          href="/pwa_icons/apple-splash-1792-828.jpg"
+          media="(device-width: 414px) and (device-height: 896px) and (-webkit-device-pixel-ratio: 2) and (orientation: landscape)"></link>
+        <link
+          rel="apple-touch-startup-image"
+          href="/pwa_icons/apple-splash-1242-2208.jpg"
+          media="(device-width: 414px) and (device-height: 736px) and (-webkit-device-pixel-ratio: 3) and (orientation: portrait)"></link>
+        <link
+          rel="apple-touch-startup-image"
+          href="/pwa_icons/apple-splash-2208-1242.jpg"
+          media="(device-width: 414px) and (device-height: 736px) and (-webkit-device-pixel-ratio: 3) and (orientation: landscape)"></link>
+        <link
+          rel="apple-touch-startup-image"
+          href="/pwa_icons/apple-splash-750-1334.jpg"
+          media="(device-width: 375px) and (device-height: 667px) and (-webkit-device-pixel-ratio: 2) and (orientation: portrait)"></link>
+        <link
+          rel="apple-touch-startup-image"
+          href="/pwa_icons/apple-splash-1334-750.jpg"
+          media="(device-width: 375px) and (device-height: 667px) and (-webkit-device-pixel-ratio: 2) and (orientation: landscape)"></link>
+        <link
+          rel="apple-touch-startup-image"
+          href="/pwa_icons/apple-splash-640-1136.jpg"
+          media="(device-width: 320px) and (device-height: 568px) and (-webkit-device-pixel-ratio: 2) and (orientation: portrait)"></link>
+        <link
+          rel="apple-touch-startup-image"
+          href="/pwa_icons/apple-splash-1136-640.jpg"
+          media="(device-width: 320px) and (device-height: 568px) and (-webkit-device-pixel-ratio: 2) and (orientation: landscape)"></link>
+      </Head>
+      <GoogleOAuthProvider
```

Comment:
Why only this context provider has not moved in Layout.tsx ?

Reply:
I prefer to let this context on top of the app with the <Analytics /> too for more clarity